doc/manual.md

```diff
@@ -109,6 +108,126 @@ install the concepts tool, simply run:
 If this process completes successfully, the tool will now be installed
 and ready to be used.
 
+Writing concepts
+=========================================
+
+In this section, we will discuss how to write a concepts file. We will
+use an example and go through it based on lines, explaining what is
+necessary. For this example we will be using the same as example as in
+previous sections, from the examples included with the concepts repo.
+This is “*Celement_with_env_1.hs*”.
+
+Concepts file layout
+--------------------
+
+```haskell
+
+	module Concept where
+
+	import Tuura.Concept.STG
+
+	circuit a b c = interface <> outputRise <> inputFall <> outputFall <> inputRise <> initialState
+	  where
+    	interface = inputs [a, b] <> outputs [c]
+
+	    outputRise = rise a ~> rise c <> rise b ~> rise c
+
+    	inputFall = rise c ~> fall a <> rise c ~> fall b
+
+	    outputFall = fall a ~> fall c <> fall b ~> fall c
+
+    	inputRise = fall c ~> rise a <> fall c ~> rise b
+
+	    initialState = initialise a False <> initialise b False <> initialise c False
+```
+
+The concepts file we will discuss is found in
+image above. The following describes important
+information about specific lines.
+
+Line 1
+
+This line must be included in all concept files, as the first line.
+This ensures that when translating, this is recognised as a
+concepts file.
+
+Line 2
+
+This must also remain in all concept files, before any concepts begin to
+be defined. Importing this module means that the standard operators
+and existing gates/protocols can be used.
+
+Line 3
+
+This is where a user can begin to define their concepts. “`circuit`” must
```

Comment:
I think backticks are enough to signify a code snippet: `circuit`. No need for adding quotes around that.